src/main/main.c

```diff
@@ -400,7 +400,7 @@ void init(void)
             i2cInit(I2C_DEVICE);
         }
     }
-#elif defined(CC3D)
+#elif defined(I2C_DEVICE_SHARES_UART3)
```

Comment:
@digitalentity @martinbudden that's my idea of removing some target dependencies in main.c